addons/account/i18n/account.pot

```diff
@@ -6410,12 +6442,6 @@ msgid "Put Money In"
 msgstr ""
 
 #. module: account
-#: selection:account.tax,amount_type:0
-#: selection:account.tax.template,amount_type:0
-msgid "Python Code"
```

Comment:
This one should stay. [it's here](https://github.com/odoo/odoo/blob/6327cda0a79a6cbae1d5f9981aed27d26cdad4fe/addons/account_tax_python/account_tax_python.py#L60). Try the export on db-all